dask/bag/tests/test_random.py

```diff
@@ -51,6 +51,20 @@ def test_choices_k_equal_bag_size_with_unbalanced_partitions():
     assert all(i in seq for i in li)
 
 
+def test_choices_with_more_bag_partitons():
+    # test with npartitions > split_every (split_every=8 by default)
+    seq = range(100)
+    sut = db.from_sequence(seq, npartitions=10)
+    random.sample(sut, k=10).compute()
+
+
+def test_sample_with_more_bag_partitons():
+    # test with npartitions > split_every (split_every=8 by default)
+    seq = range(100)
+    sut = db.from_sequence(seq, npartitions=10)
+    random.sample(sut, k=10).compute()
```

Comment:
It's great that this `compute()` now doesn't crash. Are there additional `assert`s we can add here to make sure the result of the computation is what we expect? For example, looking at the other `test_choice_*` tests, they all doing something along the lines of 

https://github.com/dask/dask/blob/f30ca4af3bc4c91aed50082154cb25cd2672e33b/dask/bag/tests/test_random.py#L39-L42

Reply:
All the tests for `sample` and `choice` were very specific that's why we checked only the compute case, but I agree that the additional asserts are a net positive, I'll add them!